packages/client/components/ActivityLibrary/ActivityDetails.tsx

```diff
@@ -156,115 +158,166 @@ const ActivityDetails = (props: Props) => {
-                        </div>
+                      <b>Reflect</b> on what’s working or not on your team. <b>Group</b> common
+                      themes and vote on the hottest topics. As you <b>discuss topics</b>, create{' '}
+                      <b>takeaway tasks</b> that can be integrated with your backlog.
+                    </div>
+                    <div className='mt-[18px] flex min-w-max items-center'>
+                      <div className='flex items-center gap-3'>
+                        <JiraSVG />
+                        <GitHubSVG />
+                        <JiraServerSVG />
+                        <GitLabSVG />
+                        <AzureDevOpsSVG />
                       </div>
-                    )}
-                  </div>
-                  <b>Reflect</b> on what’s working or not on your team. <b>Group</b> common themes
-                  and vote on the hottest topics. As you <b>discuss topics</b>, create{' '}
-                  <b>takeaway tasks</b> that can be integrated with your backlog.
-                </div>
-                <div className='mt-[18px] flex min-w-max items-center'>
-                  <div className='flex items-center gap-3'>
-                    <JiraSVG />
-                    <GitHubSVG />
-                    <JiraServerSVG />
-                    <GitLabSVG />
-                    <AzureDevOpsSVG />
-                  </div>
-                  <div className='ml-4'>
-                    <b>Tip:</b> push takeaway tasks to your backlog
+                      <div className='ml-4'>
+                        <b>Tip:</b> push takeaway tasks to your backlog
+                      </div>
+                    </div>
                   </div>
+                  <TemplatePromptList
+                    isOwner={isOwner && isEditing}
+                    prompts={prompts!}
+                    templateId={templateId}
+                  />
+                  {isOwner && isEditing && (
+                    <AddTemplatePrompt templateId={templateId} prompts={prompts!} />
+                  )}
                 </div>
               </div>
-              <TemplatePromptList isOwner={isOwner} prompts={prompts!} templateId={templateId} />
-              {isOwner && <AddTemplatePrompt templateId={templateId} prompts={prompts!} />}
             </div>
           </div>
+          <ActivityDetailsSidebar
+            selectedTemplateRef={selectedTemplate}
+            teamsRef={teams}
+            isOpen={!isEditing}
+          />
         </div>
-        <ActivityDetailsSidebar selectedTemplateRef={selectedTemplate} teamsRef={teams} />
+        {isEditing && (
+          <div className='absolute bottom-0 flex h-20 w-full items-center justify-center bg-slate-200'>
+            <button
+              onClick={() => setIsEditing(false)}
+              className='w-max cursor-pointer rounded-full bg-sky-500 px-10 py-3 text-center font-sans text-base text-lg font-semibold text-white hover:bg-sky-600'
```

Comment:
-1 you're using two different utility classes for the same css prop here `text-base text-lg`

Reply:
Oops, nice catch!